packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -159,3 +159,69 @@ class TapSemanticEvent extends SemanticsEvent {
   @override
   Map<String, dynamic> getDataMap() => const <String, dynamic>{};
 }
+
+/// An event to move the accessibility focus.
+///
+/// Using this API is generally not recommended, as it may break the consistency of the
```

Comment:
It's not clear to me what the "consistency of the accessibility focus" is. This makes it sound like you may not be able to move focus anymore after using this? Which hopefully, is not the case. What do you actually mean by consistency? Maybe, that it may break a users' expectation of how a11y focus works and therefore should be just very carefully?